pennylane_lightning/core/src/simulators/lightning_gpu/bindings/LGPUBindingsMPI.hpp

```diff
@@ -84,8 +84,25 @@ void registerBackendClassSpecificBindingsMPI(PyClass &pyclass) {
                 return new StateVectorT(devtag_local, mpi_buf_size,
                                         num_global_qubits, num_local_qubits);
             })) // qubits, device
+        .def(
```

Comment:
Does `resetState()` do the same job?

Reply:
You are completely right. I have not been able to catch this one! Thank you!